src/services/pagination.ts

```diff
@@ -0,0 +1,66 @@
+import Place from './place';
+
+export type ListPage = {
+  page: number;
+  pageSize: number;
+  totalPlaces: number;
+  totalPages: number;
+  pagePlaces: Place[];
+};
+
+export default class Pagination {
+  private page;
+  private pageSize;
+  private cookiePageSize;
+  private cookieCurrentPage;
+  private requestContactTypeName;
+
+  constructor(options: { page: number; pageSize?: number; cookie: {}; requestContactTypeName?: string }) {
+    this.pageSize = options.pageSize ? options.pageSize : 10;
+    this.page = options.page;
+    this.cookiePageSize = this.getPaginationCookie(options.cookie, 'pageSize');
```

Comment:
i want to persist page number in cookie to handle all miscellaneous call to root "/" e.g when you move out to create more users/edit etc you should come back to your "current page"
i can create logic to handle the situation you are talking about

Reply:
> i can create logic to handle the situation you are talking about

Yeah but that's just more code. Personally, I like less code. Having the state in the URL for both the current tab and page is much simpler and easier to understand at a glance both on the UI and in code

> i want to persist page number in cookie to handle all miscellaneous call to root "/"

Then maybe we should fix that instead?

Reply:
whats is much code? :) 
you want us to handle all calls to "/"? that sounds like more code than just handling this situation with a variable and an if

Reply:
cookies are good to store and persist user preferences there is nothing sensitive we are storing here that the user cant see on the ui

Reply:
> i want to persist page number in cookie to handle all miscellaneous call to root "/"
> 
> you want us to handle all calls to "/"?

Find these calls and replace them with appropriate ones i.e redirecting to the correct page. Calling `/` worked before because there was no pagination

> cookies are good for storing and persisting user preferences besides there is nothing sensitive we are storing here that the user cant see on the ui

The current page and active tab are not user preferences, they are part of the navigation state that should be on the URL

Reply:
cookies have been removed